test/testdata/infer/attached_class_reopen.rb

```diff
@@ -0,0 +1,28 @@
+# typed: true
+
+class A
+  extend T::Sig
+  extend T::Generic
+
+  X = type_member(fixed: Integer)
+
+  sig {returns(X)}
+  def test_X
+    10
+  end
+end
+
+class B; end
```

Comment:
What's this guy doing here?

Reply:
Just any statement to have between the two definition sites of the `A` class.